scripts/infra/cloud-instance.sh

```diff
@@ -13,6 +13,17 @@ else
     exit 1
 fi
 
+ensure_system_pkgs() {
+    # shellcheck source=/etc/os-release
+    source /etc/os-release
```

Comment:
This script is currently focused on operating on the cloud instance.  I am hesitant to also have it work on your local env.  Maybe this should be a sibling script in infra?

Reply:
I'm open to that for sure! I can separate this out in a followup commit